_presentations/scipy2016.md

```diff
@@ -292,7 +293,7 @@ Also a python OpenGL binding, but simplifies windowing and multimedia
 ## A Helper for Pyglet
 
 - …Write geometric shape primitives (sphere, box, etc) as "*objects*"
-- …Write mathematical primitives (vector, vertex, etc) as "*utils*"
+- …Write mathematical primitives (vector, vertex, etc)  as "*utils*"
```

Comment:
Maybe add subpackage (i.e. *utils subpackage*)? When I first read this slide I though "object" mean Python object, and wasn't sure what utils meant.